tox.ini

```diff
@@ -15,18 +15,28 @@ commands =
     pip install -e .
     py.test {posargs:pymemcache/test/ -m integration}
 
-[testenv:py27-flake8]
+[testenv:py2.7-flake8]
```

Comment:
I added a dot (`.`) to allow us to rely on `matrix.python-version`

Reply:
I seems like there's a way to map these via a `[gh-actions]` section in `tox.ini`: https://github.com/ymyzk/tox-gh-actions#factor-conditional-settings-python-version